components/post_view/post_info/post_info.jsx

```diff
@@ -17,6 +18,7 @@ import EmojiPickerOverlay from 'components/emoji_picker/emoji_picker_overlay.jsx
 import PostFlagIcon from 'components/post_view/post_flag_icon.jsx';
 import PostTime from 'components/post_view/post_time.jsx';
 import EmojiIcon from 'components/svg/emoji_icon';
+import ChannelPermissionGate from 'components/permissions_gates/channel_permission_gate';
```

Comment:
Can't find this component somewhere. Is this on master or other branch?

Reply:
Yes, this is part of the new permissions system, is in the advanced-permissions-phase-1 feature branch.

Reply:
I see. I have not seen earlier that you're submitting this against `mattermost:advanced-permissions-phase-1`